src/main.ts

```diff
@@ -377,6 +384,24 @@ getResultsButton.forEach((getResult) => {
   });
 });
 
+// event listener for animation inputs to display animation preview
+const attribute = 'animation';
+
+const getDegreeElement = getRange(attribute);
+const getRadioButtonSetElement = getRadioButtonSet(attribute);
+const getDurationElement = getInputSpinner(attribute);
+
+getDegreeElement?.addEventListener('change', () => displayAnimationPreview());
+getRadioButtonSetElement.forEach((radioButton: HTMLInputElement) => {
+  radioButton.onclick = () => {
+    displayAnimationPreview();
+  };
+});
+getDurationElement?.addEventListener('change', () => {
+  displayAnimationPreview();
+});
```

Comment:
I need you to please follow the style of the projects, if you noticed, all variables are at the top, all event listeners are grouped together and all loops are grouped together